enable/colors.py

```diff
@@ -10,358 +10,36 @@
+if toolkit.toolkit == "wx":
     from traitsui.wx.constants import WindowColor
 
-    color_table["sys_window"] = (
+    color_table["syswindow"] = (
         WindowColor.Red() / 255.0,
         WindowColor.Green() / 255.0,
         WindowColor.Blue() / 255.0,
         1.0,
     )
-
-    class ColorEditorFactory(StandardColorEditorFactory):
-        def to_wx_color(self, editor):
-            if self.mapped:
-                retval = getattr(editor.object, editor.name + "_")
-            else:
-                retval = getattr(editor.object, editor.name)
-            if isinstance(retval, tuple):
-                retval = wx.Colour(
-                    int(255 * retval[0]),
-                    int(255 * retval[1]),
-                    int(255 * retval[2]),
-                )
-            return retval
-
-        def from_wx_color(self, color):
-            """ Gets the application equivalent of a wxPython value.
-            """
-            return convert_from_wx_color(self, "color", color)
-
-        def str_color(self, color):
-            if isinstance(color, (wx.Colour, ColourPtr)):
-                return "(%d,%d,%d)" % (
-                    color.Red(),
-                    color.Green(),
-                    color.Blue(),
-                )
-            elif isinstance(color, tuple):
-                fmt = "(" + ",".join(["%0.3f"] * len(color)) + ")"
-                return fmt % color
-            return color
-
-    ColorTrait = Trait(
-        "black",
-        Tuple,
-        List,
-        color_table,
-        convert_from_wx_color,
-        editor=ColorEditorFactory,
-    )
-
-elif traits_toolkit().toolkit.startswith("qt"):
+elif toolkit.toolkit.startswith("qt"):
     from pyface.qt import QtGui
-    from traitsui.qt4.color_editor import (
-        ToolkitEditorFactory as StandardColorEditorFactory,
-    )
-
-    def convert_from_pyqt_color(obj, name, value):
-        if isinstance(value, QtGui.QColor):
-            return value.getRgbF()
-        elif type(value) is int:
-            num = int(value)
-            return (
-                (num >> 16) / 255.0,
-                ((num >> 8) & 0xFF) / 255.0,
-                (num & 0xFF) / 255.0,
-                1.0,
-            )
-        elif type(value) in (list, tuple):
-            if len(value) == 3:
-                return (
-                    value[0] / 255.0,
-                    value[1] / 255.0,
-                    value[2] / 255.0,
-                    1.0,
-                )
-            elif len(value) == 4:
-                return value
-            else:
-                raise TraitError
-        else:
-            raise TraitError
-
-    convert_from_pyqt_color.info = (
-        "a QtGui.Color instance, an SVG color "
-        "name, an integer which in hex is of the form 0xRRGGBB, where RR "
-        "is red, GG is green, and BB is blue, a list/tuple of (r,g,b) or "
-        "(r,g,b,a)"
-    )
 
+    # XXX to-do: this will not work if dark mode changes
```

Comment:
```suggestion
    # XXX to-do: this will not work if dark mode changes (see #923)
```

Reply:
I suggest the format enthought/enable#923, for ease of later searchability.